src/types/DTOs.ts

```diff
@@ -0,0 +1,17 @@
+interface CardDTO {

```

Comment:
Javascript/Typescript files may not contain upper case letters and must be written in kebab-case (an-example-file.ts) (https://github.com/Gamify-IT/docs/blob/main/dev-manuals/languages/typescript/file-names.md).